com.palantir.typescript/src/com/palantir/typescript/preferences/ProjectPreferenceStore.java

```diff
@@ -202,6 +251,186 @@ private synchronized void insertValue(String name) {
         }
     }
 
+    public boolean reloadTsConfigFile() {
```

Comment:
Yes, I thought about it, not vital but I will try to propose something here. Thanks